templates/admin/job_template/index.html.ep

```diff
@@ -124,29 +127,37 @@ scenarios:
     <i>medium</i>-<b>ARCH</b>:
     - <b>TESTSUITE</b>
                         </code>
-                        <div>Override machine, priority or settings from defaults:</div>
-                        <code>
+                        <div>Override machine, priority, settings or description:</div>
+<code>scenarios:
+  ARCH:
+    medium-ARCH:
```

Comment:
I think it was fine like it was before to not repeat the header. What is your intention here? Or is this what you mean with "always include the root in the examples" in the commit message?

Reply:
Yes, it is what I mean in the commit message. The additional lines aren't using that much space in this way it is clear where it belongs.